### PR TITLE
When creating touch events, always make sure they bubble

### PR DESCRIPTION
**Pull request by azakus**

### CODE REVIEW

File: mock-interactions.js [hunk 1]
```diff
@@ -91,10 +91,12 @@
       touches: touches,
       targetTouches: touches,
       changedTouches: touches
-    };
+    }
```
Comment (cdata) [inline mock-interactions.js]: How dare you sir... 🎩

**Comment by cdata** [comment]
We have a nominal test suite for these helpers, but it won't really tell us if this will cause problems for anyone.

The change seems good to me. If it hurts anyone, we probably have no way of knowing until it happens...

**Comment by notwaldorf** [comment]
@cdata we could in theory run tattoo and see if no other elements test fail, right?
This is LGTM though

**Comment by azakus** [comment]
Agreed. I think this change is rather important though, as it makes Chrome consistent with other browsers without constructible TouchEvents: https://github.com/PolymerElements/iron-test-helpers/blob/cae104429f4222ced6d1bf77555327329f768283/mock-interactions.js#L102

**Comment by cdata** [comment]
@notwaldorf yeah, that would cover us. But TBH I think if anyone has problems with fake touch events going from non-bubbling to bubbling, it's someone with more complex integration tests (e.g., someone testing a compound app component).

**Comment by notwaldorf** [comment]
:shipit: :shipit: :shipit:

**Comment by notwaldorf** [comment]
i'ma merge and release cause i'm antsy then

**Comment by azakus** [comment]
👍

**Pull request merged**